eitprocessing/features/breath_detection.py

```diff
@@ -76,13 +89,26 @@ def find_breaths(self, sequence: Sequence, continuousdata_label: str) -> Interva
         expiration, and end of expiration.
 
         Args:
-            sequence: the sequence that contains the data
-            continuousdata_label: label of the continuous data to apply the algorithm to
+            container: a ContinuousData object that contains the data or a Sequence object
+                that that contains the continuous data
```

Comment:
could it not be from an `EITData` object?

Reply:
Not for now. @JulietteFrancovich has been working on detecting breaths in individual pixels, but that still needs some work. For now, detection in ContinuousData, where global impedance values are stored, is supported.